frontend/src/components/resourceQuota/Details.tsx

```diff
@@ -27,11 +28,21 @@ export default function ResourceQuotaDetails() {
                   },
                   {
                     label: t('resourceQuota|Used'),
-                    getter: item => item.used,
+                    getter: item =>
+                      compareUnits(item.used, normalizeUnit(item.name, item.used)) ? (
+                        <>{normalizeUnit(item.name, item.used)}</>
+                      ) : (
+                        <>{`${item.used} (${normalizeUnit(item.name, item.used)})`}</>
+                      ),
```

Comment:
This code can be made more readable by having
```typescript
getter: item => {
  const normalizedUnits = normalizeUnit(item.name, item.used);
  ...

  return ...
}
```

And I don't think you need `<></>` as it looks like there's only one child.

Ditto for the other changes.